README.md

```diff
@@ -162,17 +170,31 @@ An anonymous file descriptor for the shared memory mapping will be created using
 
 ##### Requirements
 
-Required packages for tests (Linux-only yet):
+IPC API on Linux requires the `PTRACE_MODE_ATTACH_REALCREDS` permission (see `ptrace(2)`)
+to duplicate an another process's file descriptor (see above).
+
+Packages required for tests (Linux-only yet):
    - libnuma-dev
 
 #### Level Zero memory provider
 
 A memory provider that provides memory from L0 device.
 
+IPC API uses the file descriptor duplication. It requires using `pidfd_getfd(2)` to obtain
+a duplicate of another process's file descriptor (`pidfd_getfd(2)` is supported since Linux 5.6).
+Permission to duplicate another process's file descriptor is governed by a ptrace access mode
+`PTRACE_MODE_ATTACH_REALCREDS` check (see `ptrace(2)`) that can be changed using
+the `/proc/sys/kernel/yama/ptrace_scope` interface in the following way:
+```sh
+$ sudo bash -c "echo 0 > /proc/sys/kernel/yama/ptrace_scope"
+```
+
 ##### Requirements
 
 1) Linux or Windows OS
 2) The `UMF_BUILD_LEVEL_ZERO_PROVIDER` option turned `ON` (by default)
+3) IPC API on Linux requires the `PTRACE_MODE_ATTACH_REALCREDS` permission (see `ptrace(2)`)
```

Comment:
if this is no. 3 now, the list below should start from 4)

Reply:
Done